crates/database/src/postgres/migrations/V1__initial.sql

```diff
@@ -143,8 +143,8 @@ CREATE TABLE known_validators (
 
 
 CREATE TABLE proposer_duties (
-  "public_key" bytea PRIMARY KEY,
-  "slot_number" integer,
+  "public_key" bytea,
+  "slot_number" integer PRIMARY KEY,
```

Comment:
Why did we change the `slot_number` to primary key?

Reply:
Related helix issue: https://github.com/gattaca-com/helix/pull/43

Reply:
Actually according to comments on the PR we should revert the changes in this file and only keep the file `V22_proposer_duties_fix.sql`.

Reply:
Done